app/public/styles/variables/base.scss

```diff
@@ -2,7 +2,7 @@
 $base-font-px: 18;
 
 // Base Font Family
-$base-font-family: "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
+$base-font-family: "RedHatText", "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
```

Comment:
@ReeseSThomp We will need to add `$base-font-family--heading` now too since there are different font families for different types.  I or you might also need to go through and update our styles to use the right variable on the right class.